### PR TITLE
Feature/fungi forecast

### PR DESCRIPTION
**Pull request by rosenbach**

### CODE REVIEW

**Comment by rosenbach** [comment]
test

File: FungiJournal.API/APIClients/BayernAtlasAPIClient.cs [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
         {
             WMSRequest request = new()
             {
                 Request = "GetFeatureInfo",
                 BBox =
-                $"{lat},{lng},{lat+0.34},{lng+0.37}"
+                $"{lat},{lng},{lat + 0.34},{lng + 0.37}"
             };
 
             HttpClient httpClient = new();
```
Comment (Izelude) [inline FungiJournal.API/APIClients/BayernAtlasAPIClient.cs:18]: HttpClient is IDisposable, so it needs `using` in front of it.

File: FungiJournal.API/APIClients/BayernAtlasAPIClient.cs [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
 
             GetFeatureInfoResponse result = await httpClient.GetFromJsonAsync<GetFeatureInfoResponse>(request.Uri);
-            
-            return (result.Features.Length>0);
+
+            return (result.Features.Length > 0);
```
Comment (Izelude) [inline FungiJournal.API/APIClients/BayernAtlasAPIClient.cs:22]: `result.Features.Length.Any()`

File: FungiJournal.API/APIClients/BayernAtlasAPIClient.cs (hunk 1 above)
Comment (Izelude) [inline FungiJournal.API/APIClients/BayernAtlasAPIClient.cs:18]: HttpClient is IDisposable, therefore it needs a `using` in front of it.

Comment (rosenbach) [reply FungiJournal.API/APIClients/BayernAtlasAPIClient.cs:18]: solved via injection

Comment (rosenbach) [reply FungiJournal.API/APIClients/BayernAtlasAPIClient.cs:18]: solved via injection

**Pull request merged**